<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
    <line>
      <![CDATA[<div class="test-message">&emsp;Validating amort templaate for US networks: 1.0_Acquired Movies_Acquired MiniSeries_Acquired Movies</div>]]>
    </line>
    <line>
      <![CDATA[<div class="test-message">&emsp;launchAppUsingNativeWindowHandle: Launching app using window handle</div>]]>
    </line>
    <line>
      <![CDATA[<div class="test-message">&emsp;getElement: By: By.name: [PROJECTSTG] - COMPASS</div>]]>
    </line>
    <line>
      <![CDATA[<div class="test-message">&emsp;loginCompass: Logging in application for user: 206534643</div>]]>
    </line>
    <line>
      <![CDATA[<div class="test-result"><br><font color="green"><strong> User [206534643] successfully logged in Compass application </strong></font> </div>&emsp;]]>
    </line>
    <line>
      <![CDATA[<div class="test-message">&emsp;createContract: distributor: 10X10 Entertainment, dealType: Cash, negotiatedBy: LEGAL</div>]]>
    </line>
    <line>
      <![CDATA[<div class="test-message">&emsp;writeResultInTxtFile: writing results in txt file: 1	Acquired Movies	Acquired MiniSeries	Acquired Movies	Fail</div>]]>
    </line>
    <line>
      <![CDATA[<div class="test-message">&emsp;killApp: taskkill /F /IM NBCU.Compass.exe</div>]]>
    </line>
    <line>
      <![CDATA[<!--FAIL-->]]>
    </line>
    <line>
      <![CDATA[<div class="test-result"><br><font color="red"><strong> An unknown error occurred in the remote end while processing the command. (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 390 milliseconds
Build info: version: '3.7.1', revision: '8a0099a', time: '2017-11-06T21:01:39.354Z'
System info: host: 'BSLGDC-L2045', ip: '100.122.39.127', os.name: 'Windows 7', os.arch: 'x86', os.version: '6.1', java.version: '1.8.0_131'
Driver info: io.appium.java_client.windows.WindowsDriver
Capabilities [{platformName=Windows, appTopLevelWindow=2b1092, platform=ANY}]
Session ID: A0D899F4-DF88-411E-BF0C-E7CA455DCB18 </strong></font> </div>&emsp;]]>
    </line>
  </reporter-output>
  <suite name="Default suite" duration-ms="69671" started-at="2018-02-09T10:03:15Z" finished-at="2018-02-09T10:04:25Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="69671" started-at="2018-02-09T10:03:15Z" finished-at="2018-02-09T10:04:25Z">
      <class name="nbcu.compass.amorttemplate.test.AmortTest">
        <test-method status="PASS" signature="init()[pri:0, instance:nbcu.compass.amorttemplate.test.AmortTest@1c20538]" name="init" is-config="true" duration-ms="2215" started-at="2018-02-09T15:33:13Z" finished-at="2018-02-09T15:33:15Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method status="FAIL" signature="testAmortTemplateUS(java.lang.String)[pri:1, instance:nbcu.compass.amorttemplate.test.AmortTest@1c20538]" name="testAmortTemplateUS" duration-ms="69469" started-at="2018-02-09T15:33:16Z" description="Validating amort template for US networks" data-provider="amortDataProvider" finished-at="2018-02-09T15:34:25Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[1.0_Acquired Movies_Acquired MiniSeries_Acquired Movies]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[An unknown error occurred in the remote end while processing the command. (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 390 milliseconds
Build info: version: '3.7.1', revision: '8a0099a', time: '2017-11-06T21:01:39.354Z'
System info: host: 'BSLGDC-L2045', ip: '100.122.39.127', os.name: 'Windows 7', os.arch: 'x86', os.version: '6.1', java.version: '1.8.0_131'
Driver info: io.appium.java_client.windows.WindowsDriver
Capabilities [{platformName=Windows, appTopLevelWindow=2b1092, platform=ANY}]
Session ID: A0D899F4-DF88-411E-BF0C-E7CA455DCB18]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: An unknown error occurred in the remote end while processing the command. (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 390 milliseconds
Build info: version: '3.7.1', revision: '8a0099a', time: '2017-11-06T21:01:39.354Z'
System info: host: 'BSLGDC-L2045', ip: '100.122.39.127', os.name: 'Windows 7', os.arch: 'x86', os.version: '6.1', java.version: '1.8.0_131'
Driver info: io.appium.java_client.windows.WindowsDriver
Capabilities [{platformName=Windows, appTopLevelWindow=2b1092, platform=ANY}]
Session ID: A0D899F4-DF88-411E-BF0C-E7CA455DCB18
	at org.testng.Assert.fail(Assert.java:93)
	at nbcu.compass.amorttemplate.util.Log.fail(Log.java:420)
	at nbcu.compass.amorttemplate.test.AmortTest.testAmortTemplateUS(AmortTest.java:101)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[<div class="test-message">&emsp;Validating amort templaate for US networks: 1.0_Acquired Movies_Acquired MiniSeries_Acquired Movies</div>]]>
            </line>
            <line>
              <![CDATA[<div class="test-message">&emsp;launchAppUsingNativeWindowHandle: Launching app using window handle</div>]]>
            </line>
            <line>
              <![CDATA[<div class="test-message">&emsp;getElement: By: By.name: [PROJECTSTG] - COMPASS</div>]]>
            </line>
            <line>
              <![CDATA[<div class="test-message">&emsp;loginCompass: Logging in application for user: 206534643</div>]]>
            </line>
            <line>
              <![CDATA[<div class="test-result"><br><font color="green"><strong> User [206534643] successfully logged in Compass application </strong></font> </div>&emsp;]]>
            </line>
            <line>
              <![CDATA[<div class="test-message">&emsp;createContract: distributor: 10X10 Entertainment, dealType: Cash, negotiatedBy: LEGAL</div>]]>
            </line>
            <line>
              <![CDATA[<div class="test-message">&emsp;writeResultInTxtFile: writing results in txt file: 1	Acquired Movies	Acquired MiniSeries	Acquired Movies	Fail</div>]]>
            </line>
            <line>
              <![CDATA[<div class="test-message">&emsp;killApp: taskkill /F /IM NBCU.Compass.exe</div>]]>
            </line>
            <line>
              <![CDATA[<!--FAIL-->]]>
            </line>
            <line>
              <![CDATA[<div class="test-result"><br><font color="red"><strong> An unknown error occurred in the remote end while processing the command. (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 390 milliseconds
Build info: version: '3.7.1', revision: '8a0099a', time: '2017-11-06T21:01:39.354Z'
System info: host: 'BSLGDC-L2045', ip: '100.122.39.127', os.name: 'Windows 7', os.arch: 'x86', os.version: '6.1', java.version: '1.8.0_131'
Driver info: io.appium.java_client.windows.WindowsDriver
Capabilities [{platformName=Windows, appTopLevelWindow=2b1092, platform=ANY}]
Session ID: A0D899F4-DF88-411E-BF0C-E7CA455DCB18 </strong></font> </div>&emsp;]]>
            </line>
          </reporter-output>
        </test-method> <!-- testAmortTemplateUS -->
      </class> <!-- nbcu.compass.amorttemplate.test.AmortTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
